lib/perl/Genome/Model/Build/MetagenomicCompositionShotgun.t

```diff
@@ -6,8 +6,5 @@ use warnings;
 use above 'Genome';
 use Test::More tests => 1;
 
-# This test was auto-generated because './Model/Build/MetagenomicCompositionShotgun.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
+# This model type is deprecated
```

Comment:
Since we're removing the Model class completely, we should also remove the Build class completely!